frontend/src/app/libs/common-components/components/input/input.component.html

```diff
@@ -0,0 +1,40 @@
+<mat-form-field 
+	appearance="outline" 
+	class="input-component"
```

Comment:
no need to add word `component` there are no special meaning, and you write more code for no matter :)